tasks/package.coffee

```diff
@@ -0,0 +1,81 @@
+###
+This file contains tasks only necessary for packaging and publishing Chosen
+###
+module.exports = (grunt) ->
+
+  grunt.config 'dom_munger',
+    latest_version:
+      src: ['public/index.html', 'public/index.proto.html', 'public/options.html']
+      options:
+        callback: ($) ->
+          $('#latest-version').text(grunt.config.get('version_tag'))
+
+  grunt.config 'zip',
+    chosen:
+      cwd: 'public/'
+      src: ['public/**/*']
+      dest: 'chosen_<%= version_tag %>.zip'
+
+  grunt.config 'gh-pages',
+    options:
+      base: 'public',
+      message: 'Updated to new Chosen version <%= pkg.version %>'
+    src: ['**']
+
+  grunt.registerTask 'package-jquery', 'Update chosen.jquery.json', () ->
```

Comment:
@koenpunt there was an implication before the rebooted jquery plugin registry was killed. But in the meantime, the jQuery fondation decided that they don't need to maintain such registry.